book/src/installation.md

```diff
@@ -0,0 +1,24 @@
+# Installation
+
+If you're using `rustup` to install and manage you're Rust toolchains, Clippy is
+usually **already installed**. In that case you can skip this chapter and go to
+the [Usage] chapter.
+
+> Note: If you used the `minimal` profile when installing a Rust toolchain,
+> Clippy is not automatically installed.
+
+## Using Rustup
+
+If Clippy was not installed for a toolchain, it can be installed with
+
+```
+$ rustup component add clippy [--toolchain=<name>]
+```
+
+## From Source
+
+Take a look at the [Basics] chapter in the Clippy developer guide to find step
```

Comment:
I'm open to moving the install from source section to here. However this is user documentation not dev documentation, so I don't see the need of mentioning `cargo dev` at all.

For the install from source section: I intentionally left this in the developer guide to signal that this is not a recommended way of installing Clippy.

Reply:
> I don't see the need of mentioning `cargo dev` at all.

Since, I've started contributing to Clippy, we had two or three people who asked about installing from source. One of them wanted to have Clippy for a platform that required the installation from source, while the others intended to check their changes on a local crate. The reference of `cargo dev` is intended for the second user group. I'm fine if you leave it out if you don't agree. :upside_down_face:

Reply:
I count changing something in Clippy and then wanting to test it as Clippy dev, so they should look into the Clippy dev documentation. 

I really don't want to have dev tools and non-recommended installation methods in the user documentation, because that makes the impression, that we actively support that. And since we don't test those things in CI, I don't think we can safely say that we actively support it (for the general user).